tensorflow/contrib/parquet/python/ops/parquet_dataset_ops.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Parquet Dataset."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow.contrib.parquet.python.ops import parquet_op_loader  # pylint: disable=unused-import
+from tensorflow.contrib.parquet.python.ops import gen_dataset_ops
+from tensorflow.python.data.ops.dataset_ops import Dataset
+from tensorflow.python.data.util import nest
+from tensorflow.python.framework import dtypes
+from tensorflow.python.framework import ops
+from tensorflow.python.framework import tensor_shape
+
+
+class ParquetDataset(Dataset):
+  """A Parquet Dataset that reads the parquet file."""
+
+  def __init__(self, filenames, columns, output_types):
+    """Create a `ParquetDataset`.
+
+    `ParquetDataset` allows a user to read data from a parquet file.
+    For example:
+
+    ```python
+    dataset = tf.contrib.parquet.ParquetDataset(
+        "/foo/bar.parquet", [0, 1], (tf.bool, tf.int32))
+    iterator = dataset.make_one_shot_iterator()
+    next_element = iterator.get_next()
+    # Prints the rows of the result set of the column [0, 1].
+    while True:
+      try:
+        print(sess.run(next_element))
+      except tf.errors.OutOfRangeError:
+        break
+    ```
+
+    Args:
+      filenames: A `tf.string` tensor containing one or more filenames.
+      columns: A 1-D `tf.int32` tensor containing the columns to extract.
```

Comment:
nit: I think the kernel allows a scalar or a vector, either fix the docs or make it only accept a vector